stl/inc/atomic

```diff
@@ -475,14 +475,33 @@ bool __stdcall _Atomic_wait_compare_non_lock_free(
 
 #ifdef _WIN64
 inline bool __stdcall _Atomic_wait_compare_16_bytes(const void* _Storage, void* _Comparand, size_t, void*) noexcept {
+
+#ifdef _M_ARM64
+#ifdef __clang__ // TRANSITION, Clang 12
+    const auto _Dest = static_cast<const __int128*>(_Storage);

```

Comment:
I suspect line 488 won't like storing through a pointer-to-`const`:
```suggestion
    const auto _Dest = static_cast<__int128*>(_Storage);
```

Reply:
You can't `static_cast` from `const void*` to `__int128*` - you'll need an additional `const_cast` for that.